smt/surrogate_models/krg_based.py

```diff
@@ -57,9 +55,22 @@ def _initialize(self):
             desc="Correlation function type",
             types=(str),
         )
+        declare(
+            "nugget", [], types=(float, list), desc="a jitter for numerical stability"
```

Comment:
Default value missing.

10.0 * np.finfo(np.double).eps

Reply:
It is defined later if eval_noise = True